### ISSUE TITLE
Add job to trigger a build on commit

### ISSUE THREAD

**Issue by sopel** [cityindex/logsearch#300]
This has been extracted from #24 and depends on #298 and #299:

* every commit on master (and other applicable branches, if any) should trigger a build as usual 
* for the time being the job should build, test and upload all artifacts, i.e. continuous delivery is out of scope here

**Comment by sopel** [comment]
Closed as _Won't Fix_ due to project being retired to the [CityIndex Attic](https://github.com/orgs/cityindex-attic).
